.travis.yml

```
@@ -31,11 +31,19 @@ jobs:
     - stage: linting
       before_script: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.23.3
       script: ./scripts/verify.sh
+      after_success:
+        - echo "Linting check succeeded and your code changes passed in this test"
+      after_failure:
+        - echo "Linting check failed, run make lint to check it locally"
 
-    - stage: golden
+    - stage: generate
       # The golden_test.sh check if the the testdata is updated according to the current changes
       # To update the testdata use the Makefile targets `make generate-setup` then `make generate-testdata`
       script: ./golden_test.sh
+      after_success:
+        - echo "Generate check succeeded and your code changes passed in this test"
```

```suggestion
      after_success: echo "Generate check succeeded"
```

The second part of the sentence doesn't add anything.